package.json

```diff
@@ -94,7 +95,7 @@
   "jest": {
     "testRegex": "/test/.*.js$",
     "setupTestFrameworkScriptFile": "<rootDir>/test/helper.js",
-    "testPathIgnorePatterns": ["/node_modules/", "/test/helper.js", "/test/utils.js", "/test/__mocks__"]
+    "testPathIgnorePatterns": ["/node_modules/", "/test/helper.js", "/test/utils.js", "/test/gql.js", "/test/__mocks__"]
```

Comment:
I've added a `gql` function that does nothing. This is enough for prettier and our schema-based vscode extensions, so in an upcoming PR, I will add the ability to set that in the way we do for Emission.

Reply:
Nice 👌